Artesian/Artesian.SDK/Service/Query/QueryParamaters/MasQueryParamaters.cs

```diff
@@ -27,20 +28,86 @@ public MasQueryParamaters()
         /// <param name="timezone"></param>
         /// <param name="filterId"></param>
         /// <param name="products"></param>
+        /// <param name="fillerK"></param>
```

Comment:
group these into a MarketAssessmentValue class reusing an existing one (should be there).